src/appPackageManifest.ts

```diff
@@ -11,12 +11,18 @@ import Vinyl from 'vinyl';
 
 import { SupportedDeviceCapabilities } from './capabilities';
 import { normalizeToPOSIX } from './pathUtils';
-import ProjectConfiguration from './ProjectConfiguration';
+import { ProjectConfiguration , AppProjectConfiguration, Tile, AppType } from './ProjectConfiguration';
 import { apiVersions } from './sdkVersion';
 
 const manifestPath = 'manifest.json';
 const PLUGIN_NAME = 'appPackageManifest';
 
+interface GalleryTile {
```

Comment:
nit: You can name this just `Tile`, the imported one is unused and it makes sense within the context of the file even though it's used elsewhere. Packages are used for more than just gallery (eg, sideloading).